chore(depcheck): explicit readme usage (#69)
lumapps/dep-check: README.md

-o / --output | The output file (yaml format) | :heavy_check_mark: | dependency_config.yaml
--lang | The language the project is written in | :heavy_check_mark: | python

> By default, any import is forbidden (blacklist). The configuration is a whitelist.

Reviewer: ```suggestion
By default, any import is forbidden (denylist). The configuration is an allowlist.
```
